tests/cases/trait/one-per-file

```diff
@@ -0,0 +1,20 @@
+---DESCRIPTION---
+File must only contain one trait
+---FILENAME---
+Foo.php
+---CONTENTS---
+<?php
+
+namespace Vendor;
+
+trait Foo
+{
+}
+
+trait Bar
+{
+}
+
+---MESSAGES---
+9:1 PSR1.Classes.ClassDeclaration.MultipleClasses
```

Comment:
Doesn't trigger `Squiz.Classes.ClassFileName.NoMatch` because of https://github.com/squizlabs/PHP_CodeSniffer/pull/2140 (and why there's not yet a `file-name` test for traits).